packager/file/http_file_unittest.cc

```diff
@@ -165,21 +214,35 @@ TEST(HttpFileTest, MultipleWrites) {
 
   ASSERT_EQ(file->Write(data1.data(), data1.size()),
             static_cast<int64_t>(data1.size()));
+  ASSERT_TRUE(file->Flush());
```

Comment:
Why do we call Flush after every write in this test? Is it needed after every write?

Reply:
It adds coverage for the change in Flush() semantics from #1201.  Now that Flush() only flushes, this shows the new behavior.  Before #1201, you could not Write() after Flush().

In the test server, you can see those pieces written as separate chunks now.  This isn't used yet, but it may be an important detail for chunked uploads for LL streaming in the future.

Reply:
I see. It is probably better to add another MultipleWritesWithFlush to test this case. Otherwise consecutive Writes without Flush is not tested.

Reply:
Good point.

Reply:
Done